ui/app/templates/components/oidc/client-form.hbs

```diff
@@ -0,0 +1,91 @@
+<PageHeader as |p|>
+  <p.top>
+    <nav class="breadcrumb">
+      <ul>
+        <li>
+          <span class="sep">&#x0002f;</span>
+          {{#if @model.isNew}}
+            <LinkTo @route="vault.cluster.access.oidc.clients">
+              Applications
+            </LinkTo>
+          {{else}}
+            <LinkTo @route="vault.cluster.access.oidc.clients.client.details" @model={{@model.name}}>
+              Details
+            </LinkTo>
+          {{/if}}
+        </li>
+      </ul>
+    </nav>
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3">
+      {{if @model.isNew "Create" "Edit"}}
+      application
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+<form {{on "submit" (perform this.save)}}>
+  <div class="box is-sideless is-fullwidth is-bottomless">
+    <MessageError @errorMessage={{this.error}} class="has-top-margin-s" />
+    {{#each @model.formFields as |attr|}}
+      <FormField data-test-field={{true}} @attr={{attr}} @model={{@model}} @modelValidations={{this.modelValidations}} />
+    {{/each}}
+
+    {{! MORE OPTIONS TOGGLE }}
+    <FormFieldGroups @renderGroup="More options" @model={{@model}} @modelValidations={{this.modelValidations}} />
+  </div>
+  {{! RADIO CARD + SEARCH SELECT }}
+  <div class="box is-sideless is-fullwidth is-marginless has-top-padding-xxl">
+    {{#if @model.assignments.isFulfilled}}
+      <h4 class="title is-4">Assign access</h4>
+      <div class="is-flex-row">
+        <RadioCard
+          @title="Allow everyone to access existing"
+          @description="All Vault entities can authenticate through this application."
+          @icon="org"
+          @value="allow_all"
+          @groupValue={{this.radioCardGroupValue}}
+          @onChange={{fn (mut this.radioCardGroupValue)}}
+        />
+        <RadioCard
+          @title="Limit access to selected users"
+          @description="Choose or create an assignment to give access to selected entities."
+          @icon="users"
+          @value="limited"
+          @groupValue={{this.radioCardGroupValue}}
+          @onChange={{fn (mut this.radioCardGroupValue)}}
+        />
+      </div>
+      {{#if (eq this.radioCardGroupValue "limited")}}
+        <SearchSelectWithModal
+          @id="assignments"
+          @label="assignment"
+          @labelClass="is-label"
```

Comment:
This seems like something superfluous we could assume in the SearchSelectWithModal template and remove from the params